src/ray/protobuf/autoscaler.proto

```diff
@@ -93,9 +93,22 @@ message ResourceRequest {
   repeated LabelSelector label_selectors = 3;
 }
 
+message ResourceRequestByCountBreakdown {
```

Comment:
This seems to be almost identical to `message ResourceDemand`. If we split it into two separate messages, we must always ensure that both types remain consistent. Is it possible to use `ResourceDemand` instead?

https://github.com/ray-project/ray/blob/054369c400f3d88a577ff7c1b2a7fc70bc39371c/src/ray/protobuf/gcs.proto#L488-L501

Reply:
I am ok to reuse ResourceDemand tough it doesn’t seem to be a good idea because the purpose of ResourceDemand is already well defined (Represents the demand for a particular resource shape.) and it doesn’t seems to fit here well as we don’t need the shape field here.